scripts/job.star

```diff
@@ -0,0 +1,34 @@
+load (
+    'scripts/lib.star',
+    'scan_docker_image_unkown_low_medium_vulnerabilities_step',
+    'scan_docker_image_high_critical_vulnerabilities',
+)
+def cronjobs(edition):
+    if edition != 'oss':
+        edition='grafana-enterprise'
+    else:
+        edition='grafana'
+
+    trigger = {
+        'event': 'cron',
+        'cron': 'nightly',
+    }
+    platform_conf = {
+        'os': 'linux',
+        'arch': 'amd64',
+    }
+    steps=[ 
+        scan_docker_image_unkown_low_medium_vulnerabilities_step(edition),
+        scan_docker_image_high_critical_vulnerabilities(edition),
+    ]
```

Comment:
Can you just produce the steps list with a loop? That would be much less code, and I think we shouldn't add functions to scripts/lib.star that aren't used elsewhere, since it just fattens up the common library.

Reply:
yes, I moved it out of lib and added a loop

Reply:
Thanks Ida, having another look.